lib/StartInfo.h

```diff
@@ -80,6 +80,7 @@ struct DLL_LINKAGE StartInfo
 	ui32 seedToBeUsed; //0 if not sure (client requests server to decide, will be send in reply pack)
 	ui32 seedPostInit; //so we know that game is correctly synced at the start; 0 if not known yet
 	ui32 mapfileChecksum; //0 if not relevant
+	std::string uuid;

```

Comment:
we use uuid in several places, it's not clear which exact uuid is stored in StartInfo.
I recommend to rename to gameUuid or sessionUuid or something more infromative